android/src/main/kotlin/com/hypertrack/sdk/flutter/common/HyperTrackError.kt

```diff
@@ -0,0 +1,14 @@
+package com.hypertrack.sdk.flutter.common
+
+// enum naming convention is ignored to make datatype sync
+// across platforms easier
+@Suppress("EnumEntryName")
+enum class HyperTrackError {
```

Comment:
It looks like some errors are missing comparing this class to `lib/data_types/hypertrack_error.dart`

```
enum HyperTrackError {
  gpsSignalLost,
  locationMocked,
  locationPermissionsDenied,
  locationPermissionsInsufficientForBackground,
  locationPermissionsNotDetermined,
  locationPermissionsReducedAccuracy,
  locationPermissionsProvisional,
  locationPermissionsRestricted,
  locationServicesDisabled,
  locationServicesUnavailable,
  motionActivityPermissionsNotDetermined,
  motionActivityPermissionsDenied,
  motionActivityServicesDisabled,
  motionActivityPermissionsRestricted,
  networkConnectionUnavailable,
  invalidPublishableKey,
  blockedFromRunning
}
```

Is this expected?

Reply:
fixed